.github/workflows/build-macos.yml

```diff
@@ -17,6 +17,8 @@ jobs:
               EXTRA_XCONFIGURE_FLAGS="CFLAGS='-arch x86_64'"
           - arch: arm64
     runs-on: macos-14
+    env:
+      GOTOOLCHAIN: local
```

Comment:
Is it expected that this change coupled with our `GO_VERSION` changes elsewhere will force the build to use the locally installed Go toolchain (i.e., 1.22.11)?

Reply:
Yep. I think this is not strictly necessary because Go seems to implement its own runtime manager these days (based on the `toolchain` line in go.mod) but I figure if I'm downloading the toolchain myself I don't want it downloading another for me. I'd rather it just break.